recipes/carapace/recipe.yaml

```diff
@@ -0,0 +1,44 @@
+context:
+  name: carapace
+  version: "1.2.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/carapace-sh/carapace-bin/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 7eebc2b3f7ac4fb5641612343ce813b2f7e1c0fffd1251255542e9bfbf9ed207
+build:
+  script:
+    - cd cmd/carapace
+    - go generate ./...
+    - go install
+    - go-licenses save . --save_path ../../library_licenses
```

Comment:
It could be an improvement to use `$SRC_DIR/library_licenses` so that it's a absolute path.